release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -729,6 +734,16 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 
 * Previously, systemd units were cleaned up only when completely removed. As a result, systemd units could not be unmasked by using a machine config because the masks were not being removed unless the systemd unit was completely removed. With this fix, when you configure a systemd unit as `mask: true` in a machine config, any existing masks are removed. As a result, systemd units can be unmasked.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966445[*BZ#1966445*])
 
+[discrete]
+[id="ocp-4-10-management-console-bug-fixes"]
+==== Management Console
+
+* Previously, the *OperatorHub* category and card links did not include valid `href` attributes. Consequently, the *OperatorHub* cateogry and card links could not be opened in a new tab. This update adds valid `href` attributes to the *OperatorHub* category and card links. As a result, the *OperatorHub* and its card links can be opened in new tabs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2013127[*BZ#2013127*])
+
+* Previously, on the *Operand Details* page, a special case was created wherein the conditions table for the `status.conditions` property always rendered before all other tables. Consequently, the `status.conditions` table did not follow the ordering rules of descriptors, which caused unexpected behavior when users tried to change the order of the tables. This update removes the special case for `status.conditions` and only defaults to rendering it first if no descriptor is defined for that property. As a result, the table for `status.condition` is rendered according to descriptor ordering rules when a descriptor is defined on that property. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2014488[*BZ#2014488*])
+
+* Previously, the *Resource Details* page metrics tab was hitting the cluster-scoped Thanos endpoint. Consequently, users without authorization for this endpoint would receive a `401` response for all queries. With this update, the Thanos tenancy endpoints are updated, and redundant namespace query arguments are removed. As a result, users with the correct role-based access control (RBAC) permissions can now see data in the metrics tab tab of the *Resource Details* page. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2015806[*BZ#2015806*])
```

Comment:
"Meeting" or "exceeding" instead of the colloquial "hitting"?

Also, "tab" repeated twice.